ScoutSuite/__main__.py

```diff
@@ -97,6 +97,7 @@ def main(passed_args=None):
                                                     key_file=args.key_file,
                                                     user_account=args.user_account,
                                                     service_account=args.service_account,
+                                                    all=args.all,
```

Comment:
let's use something more obvious such as `gcp_scan_all`

in truth would be good to prepend all provider-specific arguments with `provider_`

Reply:
Changed flag to gcp_scan_all